sympy/solvers/tests/test_solveset.py

```diff
@@ -2023,3 +2023,138 @@ def test_linear_coeffs():
     raises(ValueError, lambda:
         linear_coeffs(1/x*(x - 1) + 1/x, x))
     assert linear_coeffs(a*(x + y), x, y) == [a, a, 0]
+
+# modular tests
+def test_is_modular():
+    x, y = symbols('x y')
+
+    assert _is_modular(y, x) is False
+    assert _is_modular(Mod(x, 3) - 1, x) is True
+    assert _is_modular(Mod(x**3 - 3*x**2 - x + 1, 3) - 1, x) is True
+    assert _is_modular(Mod(exp(x + y), 3) - 2, x) is True
+    assert _is_modular(Mod(exp(x + y), 3) - log(x), x) is True
+    assert _is_modular(Mod(x, 3) - 1, y) is False
+    assert _is_modular(Mod(x, 3)**2 - 5, x) is False
+    assert _is_modular(Mod(x, 3)**2 - y, x) is False
+    assert _is_modular(exp(Mod(x, 3)) - 1, x) is False
+    assert _is_modular(Mod(3, y) - 1, y) is False
+
+
+def test_invert_modular():
+    x, y = symbols('x y')
+    n = Dummy('n')
+    from sympy.solvers.solveset import _invert_modular as invert_modulo
```

Comment:
Something minor: modular / modulo ?
It's confusing to import `_invert_modular` as `invert_modulo`.

Reply:
Okay I ll change the name.